docs/config/server-options.md

```diff
@@ -161,6 +161,8 @@ The error that appears in the Browser when the fallback happens can be ignored.
 
 :::
 
+Checkout [`vite-setup-catalogue`](https://github.com/sapphi-red/vite-setup-catalogue) for some examples.
+
```

Comment:
```suggestion
Check out [`vite-setup-catalogue`](https://github.com/sapphi-red/vite-setup-catalogue) for some examples.

```

I think we should also move this above the `:::tip` as the blockquote is usually used to convey extra info before going to the next section.